spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientConfigMapPropertySource.java

```diff
@@ -52,8 +52,8 @@ private static SourceData getSourceData(KubernetesClientConfigContext context) {
 	// we need to pass various functions because the code we are interested in
 	// is protected in ConfigMapPropertySource, and must stay that way.
 	private static KubernetesClientContextToSourceData namedConfigMap() {
```

Comment:
we do not need `ConfigMapPropertySource::withPrefix` anymore, as things are slightly more involved, but only slightly